public/style.css

```diff
@@ -1,13 +1,215 @@
-.centered {
-  position: fixed;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
+@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
```

Comment:
The app looks really nice, but I am a bit worried about the amount of css that is added here. Can we reduce it to the bare minimum?